docs/integrations/mlflow.rst

```diff
@@ -3,54 +3,91 @@
+|Run on Colab|
+
+What will you get with this integration?
+----------------------------------------
+
+|mlflow-tour|
+
+|neptune-mlflow| is an open source project curated by Neptune team that enables |mlflow| experiment runs to be hosted on Neptune.
 
 The integration lets you enjoy the best of both worlds: the tracking and reproducibility of MLflow with the organization and collaboration of Neptune.
 
+.. note::
+
+    This integration is tested with ``mlflow==1.12.1``
+	
 .. image:: ../_static/images/integrations/mlflow_neptuneml.png
    :target: ../_static/images/integrations/mlflow_neptuneml.png
    :alt: organize MLflow experiments in Neptune
 
-Check the example project in Neptune: |mlflow-integration|.
+.. _mlflow-quickstart:
 
-Installation
-------------
+Quickstart
+----------
 
-.. code-block:: bash
+This quickstart will show you how to:
+
+* Install the necessary neptune packages
+* Sync your MLruns with Neptune.
+
+|Run on Colab|
+
+.. _mlflow-before-you-start-basic:
+
+Before you start
+^^^^^^^^^^^^^^^^
+#. This integration needs you to have your Personal API token. You need a Neptune account to have this. Create one for free |neptune-register| if you haven't already
+
+#. Ensure that you have ``Python 3.x`` and following libraries installed:
 
-    pip install neptune-mlflow
+   * ``neptune-mlflow``
+   * ``mlflow==1.12.1``. See |mlflow-install|.
+   
+   .. code-block:: bash
+   	
+      pip install --quiet mlflow neptune-mlflow
 
-Sync your MLruns with Neptune
------------------------------
+#. You also need minimal familiarity with Mlflow. Have a look at the |mlflow-guide| guide to get started.
 
-1. Set your ``NEPTUNE_API_TOKEN``:
+Step 1: Set your ``NEPTUNE_API_TOKEN``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Linux/IOS:
+Linux/iOS:
 
 .. code:: bash
 
-    export NEPTUNE_API_TOKEN='YOUR_NEPTUNE_API_TOKEN'
+    export NEPTUNE_API_TOKEN='YOUR_API_TOKEN'
 
 Windows:
 
 .. code-block:: bat
 
-    set NEPTUNE_API_TOKEN="YOUR_NEPTUNE_API_TOKEN"
+    set NEPTUNE_API_TOKEN="YOUR_API_TOKEN"
 
-2. Navigate to the MLflow project in your directory and run:
+.. tip::
+
+    Read more about how to :ref:`securely set the Neptune API token <how-to-setup-api-token>`.
+
+Step 2: Sync your MLruns with Neptune
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Navigate to the MLflow project in your directory and run:
 
 .. code-block:: bash
 
     neptune mlflow --project USER_NAME/PROJECT_NAME
 
-Alternatively you can point to the MLflow project directory:
+Alternatively, you can point to the MLflow project directory:
 
 .. code-block:: bash
 
-    neptune mlflow /PATH/TO/MLflow_PROJECT --project WORKSPACE_NAME/PROJECT_NAME
+    neptune mlflow /PATH/TO/MLflow_PROJECT --project USER_NAME/PROJECT_NAME
 
 **That's it! You can now browse and collaborate on your MLflow runs in Neptune.**
 
+Check out this |mlflow-integration|.
+
 Organize and share your MLflow experiments
```

Comment:
I'd drop this header and just show a screenshot of your experiment. 
We need to update those screenshot (I'd just use one here) as the UI has changed since the last time we created it